pyvrp/Population.py

```diff
@@ -140,21 +151,40 @@ def clear(self):
         self._feas = SubPopulation(self._op, self._params)
         self._infeas = SubPopulation(self._op, self._params)
 
-    def get_binary_tournament(self, rng: XorShift128) -> Individual:
+    def get_binary_tournament(
+        self,
+        rng: XorShift128,
+        cost_evaluator: CostEvaluator,
+        update_fitness: bool = True,
+    ) -> Individual:
```

Comment:
For `update_fitness` a keyword-only argument makes sense to me. Just a True/False when calling this does not say much.

Reply:
Yes, although I followed your suggestion to remove it completely.